includes/Database_Upgrader.php

```diff
@@ -104,20 +104,7 @@ public function __construct( Injector $injector ) {
 	 * @return void
 	 */
 	public function register() {
-		if ( is_admin() ) {
-			$this->run_upgrades();
-		}
-	}
-
-	/**
-	 * Get the action priority to use for registering the service.
-	 *
-	 * @since 1.6.0
-	 *
-	 * @return int Registration action priority to use.
-	 */
-	public static function get_registration_action_priority(): int {
-		return 5;
+		add_action( 'admin_init', [ $this, 'run_upgrades' ], 5 );
```

Comment:
Is there still a good reason for priority 5?

Reply:
It used to be set to 5.

Reply:
I know, but is it worth reconsidering this?

With #8513 we have other ways to ensure that the database upgrade runs before anything else.

Reply:
More than happy to have to default of 10.